examples/pylint/R1735_use_dict_literal.py

```diff
@@ -0,0 +1,3 @@
+student_dict = dict() # Error on this line.
+
+student_dict["123456789"] = "John Doe"
```

Comment:
I'd prefer showing an example using a loop, similar to what you had for R1734 for lists. Otherwise, the code is a bit simple, and in fact this example is misleading since our approved style would be `student_dict = {"123456789": "John Doe"}`, which defeats the purpose of the check.